config/initializers/inline_svg.rb

```diff
@@ -0,0 +1,7 @@
+require Rails.root.join("lib/svg_to_symbol_transform.rb")
+require Rails.root.join("lib/replace_magenta_svg_color.rb")
```

Comment:
I'd probably just put the custom transform classes inline here rather than putting them in `lib` especially as autoloading doesn't even work here - at least I assume that's why you had to require them explicitly.

Other initialisers that do similar things just define the code they want in place. I do usually put it in `module OpenStreetMap` though, to make it clear it's a customisation if it appears in a stack trace.